src/app/taskdata.ts

```diff
@@ -0,0 +1,11 @@
+export interface taskData {
+    Title: string;
```

Comment:
Lowercase field names please 
For multiple words it would be of course like this:
createdOn